packages/calcite-components/src/components/combobox/combobox.e2e.ts

```diff
@@ -1624,13 +1624,11 @@ describe("calcite-combobox", () => {
   it("should gain focus when it's items are selected via click", async () => {
     const page = await newE2EPage();
     await page.setContent(html` <calcite-combobox id="demoId">
-      <calcite-combobox-item value="test-value" text-label="test"> </calcite-combobox-item>
+      <calcite-combobox-item id="demoItemId" value="test-value" text-label="test"> </calcite-combobox-item>
```

Comment:
Is this test update related to the `floating-ui` changes? It seems disconnected.

Reply:
Sidebar: this test seems suspect. It's asserting on combobox receiving focus when clicking on an item when closed?

Reply:
@jcfranco yeah I couldn't get this test to pass otherwise. Not sure if its related or not but your observations make sense. I can try setting it to be open?